### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by trustin**
- Jackson 2.9.3 -> 2.9.4
- Checkstyle 8.7 -> 8.8
- Dropwizard Metrics 3.2.6 -> 4.0.2
- gRPC 1.8.0 -> 1.9.0
- Prometheus 0.1.0 -> 0.2.0
- Brave 4.13.3 -> 4.14.1
- Zipkin 2.4.3 -> 2.4.5
- javax.validation 2.0.0 -> 2.0.1
- JSON-unit 1.27.0 -> 1.28.1
- Apache HTTP cient 4.5.3 -> 4.5.5
- Apache Tomcat 8.5.24 -> 8.5.27 and 8.0.48 -> 8.0.49
- AssertJ 3.8.0 -> 3.9.0
- Jetty ALPN agent 2.0.6 -> 2.0.7
- JMH 1.19 -> 1.20

### CODE REVIEW

**Comment by trustin** [comment]
Will try to update to Thrift 0.11 and Tomcat 9.0 next time.

**Comment by trustin** [comment]
BTW, Dropwizard Metrics 4 is a cleaned up, Java 8, mostly API compatible version of Metrics 3.2.

File: it/src/test/java/com/linecorp/armeria/server/grpc/interop/ArmeriaGrpcServerBuilder.java [hunk 1]
```diff
@@ -1,99 +0,0 @@
...
-import com.linecorp.armeria.server.ServiceRequestContext;
-import com.linecorp.armeria.server.grpc.GrpcServiceBuilder;
-
-import io.grpc.Server;
-import io.grpc.ServerServiceDefinition;
-import io.grpc.ServerStreamTracer.Factory;
-import io.grpc.internal.AbstractServerImplBuilder;
-import io.grpc.internal.InternalServer;
-
-public class ArmeriaGrpcServerBuilder extends AbstractServerImplBuilder<ArmeriaGrpcServerBuilder> {
```
Comment (anuraaga) [inline it/src/test/java/com/linecorp/armeria/server/grpc/interop/ArmeriaGrpcServerBuilder.java]: Woohoo!

**Comment by trustin** [comment]
Thanks!

**Pull request merged**